pybar_fei4_interpreter/defines.h

```diff
@@ -145,11 +145,15 @@ const unsigned int RAW_DATA_MAX_ROW=336;
 
 //trigger word macros
 #define TRIGGER_WORD_HEADER_MASK_NEW 0x80000000 // 1xxx xxxx xxxx xxxx xxxx xxxx xxxx xxxx, x may contain user data
-#define TRIGGER_NUMBER_MASK_NEW		0x0000FFFF //trigger number is in the low word
-#define TRIGGER_TIME_STAMP_MASK		0x7FFF0000 //trigger number is in the low word
+#define TRIGGER_NUMBER_MASK_NEW		0x7FFFFFFF //trigger number is in the low word
+#define TRIGGER_TIME_STAMP_MASK		0x7FFFFFFF //trigger number is in the low word
+#define TRIGGER_NUMBER_MASK_M26		0x0000FFFF //trigger number of M26

```

Comment:
This is not for Mimosa only, but a mode that can be generally chosen. So we should rename